backend/go-app/main.go

```diff
@@ -397,6 +397,52 @@ func checkUsername(Username string) error {
 	return nil
 }
 
+func isNoProxy(rawURL string) bool {
```

Comment:
Give it a Git related noproxy name if that what it is used for :P

Reply:
but it could be use for any proxy condition, no? To check if the url is part of NO_PROXY? Not specifically for git.

Reply:
Isn't that handled by the default library if we implement NO_PROXY with the default proxy HTTP config? 

https://cs.opensource.google/go/x/net/+/refs/tags/v0.28.0:http/httpproxy/proxy.go;l=51

Reply:
The git library does not by default but that's actually smart to just force git to use a `net/http` Client which provide us to use `http.ProxyFromEnvironment`.